main.js

```diff
@@ -0,0 +1,68 @@
+let billAmountInput = document.querySelector('.bill-amount');
+let numOfPeopleInput = document.querySelector('.num-of-people');
+let calcBtn = document.querySelector('.calc-btn');
+let clearBtn = document.querySelector('.clear-btn');
+
+window.addEventListener('load', handlePageLoad);
+billAmountInput.addEventListener('keyup', btnStatus);
+numOfPeopleInput.addEventListener('keyup', btnStatus);
+calcBtn.addEventListener('click', calculateTotal);
+clearBtn.addEventListener('click', resetInputs);
+
+function handlePageLoad() {
+  btnStatus();
```

Comment:
Perhaps it would be better to have a single boolean for keeping track of whether the buttons need to be enabled or disabled.